unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSetMethods.java

```diff
@@ -71,8 +72,11 @@ public static boolean isSet(Object baseObject, long baseOffset, int index) {
    * Returns {@code true} if any bit is set.
    */
   public static boolean anySet(Object baseObject, long baseOffset, long bitSetWidthInBytes) {
-    for (int i = 0; i <= bitSetWidthInBytes; i++) {
-      if (PlatformDependent.UNSAFE.getByte(baseObject, baseOffset + i) != 0) {
+    assert bitSetWidthInBytes % SIZE_OF_LONG == 0;
+    int widthInLong = (int)(bitSetWidthInBytes / SIZE_OF_LONG);
+    long addr = baseOffset;
+    for (int i = 0; i <= widthInLong; i++, addr += 8) {
```

Comment:
Since we have `SIZE_OF_LONG`, why we use `8` here? Or why not use `8` everywhere...